test/replay/_replay.py

```diff
@@ -0,0 +1,94 @@
+import sys, os
```

Comment:
I would like to move the script into https://github.com/eclipse/kuksa.val/tree/master/kuksa_feeders